<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>button</class>
 <widget class="QWidget" name="button">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>35</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>190</width>
    <height>35</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>190</width>
    <height>35</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>7</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="body" native="true">
     <property name="styleSheet">
      <string notr="true">#body
{
	border-radius: 5px;
}

#body:hover
{
	background-color: rgba(101,208,109,0.2);
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>8</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>22</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="icon">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>图</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="text">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#text
{
	font-size: 16px;
	font-family: 黑体;
}</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="danceLine" native="true">
        <property name="styleSheet">
         <string notr="true">QLabel
{
	background-color: green;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="line1">
           <property name="minimumSize">
            <size>
             <width>2</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>2</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="line2">
           <property name="minimumSize">
            <size>
             <width>2</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>2</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="line3">
           <property name="minimumSize">
            <size>
             <width>2</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>2</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="line4">
           <property name="minimumSize">
            <size>
             <width>2</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>2</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
